arangod/IResearch/IResearchAqlAnalyzer.h

```diff
@@ -60,9 +60,12 @@ class AqlAnalyzer final : public irs::analysis::analyzer{
     /// this could be used fo index filtering.
     bool keepNull{ true };
 
-    /// @brief  batch size for running query. Set to 1 as most of the cases
-    /// we expect just 1 to 1 modification query.
-    uint32_t batchSize{ 1 };
+    /// @brief  batch size for running query. Set to 10 as most of the cases
+    /// we expect just simple query.
+    uint32_t batchSize{ 10 };
```

Comment:
oh I think this was a misunderstanding.
BatchSize 1 here is correct. (You want one line), and afaik you pretty-much ensure that you get one line for the index right?
I was talking about the internal AQL BatchSize used by the ExecutionBlocks.
You cannot control it from here.